docker/compose/validator-testnet/docker-compose.yaml

```diff
@@ -19,7 +19,7 @@ services:
     validator:
         # Note this image currently does not support this, will update to the appropriate minimum
         # version shortly
-        image: libra/validator:master_f53da21f
+        image: libra/validator:master_609c0b05
```

Comment:
can we just use :latest ?

Reply:
`:latest` doesn't exist for `diem/validator`